pkgs/by-name/gr/grip-grab/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+let
+  pname = "grip-grab";
+  version = "0.3.0";
+in
+rustPlatform.buildRustPackage {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "alexpasmantier";
+    repo = "grip-grab";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-5nKDeJ5d9ECd9zvSK22zN15m8qCt/L3VKem9hVQzLzU=";
+  };
+
+  cargoHash = "sha256-NEoZYdZEhT4gPMm9fUn3WbFPIcjVhLLFxEvbQxVHV+k=";
```

```suggestion
  cargoHash = "sha256-g7YdNNN6YJiW8sQa1KMj5evVbLPObRXrKM5HoG87Kxc=";

  doCheck = false; # no tests
```
Test configuration takes quite some time to compile for no benefit.